usage: conda [-h] [-v] [--no-plugins] [-V] COMMAND ...
conda: error: argument COMMAND: invalid choice: 'deactivate' (choose from 'clean', 'compare', 'config', 'create', 'info', 'init', 'install', 'list', 'notices', 'package', 'remove', 'uninstall', 'rename', 'run', 'search', 'update', 'upgrade', 'build', 'content-trust', 'convert', 'debug', 'develop', 'doctor', 'index', 'inspect', 'metapackage', 'render', 'repoquery', 'skeleton', 'repo', 'token', 'verify', 'env', 'server')
A conda environment has been detected CONDA_PREFIX=
/scratch/bcyi/esenergun/knnlm 
anaconda3_gpu is loaded. Consider running conda deactivate and reloading it.
Traceback (most recent call last):
  File "/work/hdd/bcyi/eergun/knnlm-retrieval-quality/preprocess.py", line 7, in <module>
    from fairseq_cli.preprocess import cli_main
  File "/work/hdd/bcyi/eergun/knnlm-retrieval-quality/fairseq_cli/preprocess.py", line 18, in <module>
    from fairseq import options, tasks, utils
  File "/work/hdd/bcyi/eergun/knnlm-retrieval-quality/fairseq/__init__.py", line 9, in <module>
    import fairseq.criterions  # noqa
  File "/work/hdd/bcyi/eergun/knnlm-retrieval-quality/fairseq/criterions/__init__.py", line 10, in <module>
    from fairseq.criterions.fairseq_criterion import FairseqCriterion
  File "/work/hdd/bcyi/eergun/knnlm-retrieval-quality/fairseq/criterions/fairseq_criterion.py", line 10, in <module>
    from fairseq import metrics, utils
  File "/work/hdd/bcyi/eergun/knnlm-retrieval-quality/fairseq/utils.py", line 21, in <module>
    from fairseq.modules import gelu, gelu_accurate
  File "/work/hdd/bcyi/eergun/knnlm-retrieval-quality/fairseq/modules/__init__.py", line 9, in <module>
    from .character_token_embedder import CharacterTokenEmbedder
  File "/work/hdd/bcyi/eergun/knnlm-retrieval-quality/fairseq/modules/character_token_embedder.py", line 13, in <module>
    from fairseq.data import Dictionary
  File "/work/hdd/bcyi/eergun/knnlm-retrieval-quality/fairseq/data/__init__.py", line 20, in <module>
    from .indexed_dataset import IndexedCachedDataset, IndexedDataset, IndexedRawTextDataset, MMapIndexedDataset
  File "/work/hdd/bcyi/eergun/knnlm-retrieval-quality/fairseq/data/indexed_dataset.py", line 89, in <module>
    6: np.float,
  File "/sw/external/python/anaconda3_gpu/lib/python3.9/site-packages/numpy/__init__.py", line 305, in __getattr__
    raise AttributeError(__former_attrs__[attr])
AttributeError: module 'numpy' has no attribute 'float'.
`np.float` was a deprecated alias for the builtin `float`. To avoid this error in existing code, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
